bn-apps/cordapp-updates-distribution/design/design.md

```diff
@@ -55,62 +55,53 @@ The proposal is to distribute CorDapps via Maven repositories. This would allow
 
-##### cds-lib
+##### corda-updates-core
 
-`cds-lib` is a convenience wrapper around Maven Resolver. It will support full Maven coordinates, in the format of `<groupId>:<artifactId>[:<extension>[:<classifier>]]:<version>` as well as version ranges. Version ranges will be specifiable in mathematical range notation, i.e. "[1.0,2.0)", "[1.0,)" or "[1.0]". Range queries are supported by Maven Resolver out-of-the-box and will not require extra development efforts. `cds-lib` will allow to:
+`corda-updates-core` is a convenience wrapper around Maven Resolver. It will support full Maven coordinates, in the format of `<groupId>:<artifactId>[:<extension>[:<classifier>]]:<version>` as well as version ranges. Version ranges will be specifiable in mathematical range notation, i.e. "[1.0,2.0)", "[1.0,)" or "[1.0]". Range queries are supported by Maven Resolver out-of-the-box and will not require extra development efforts. `corda-updates-core` will allow to:
 * Fetch metadata about a single or a range of versions from a remote Maven repository.
-* Download a single or a range of versions from a remote repository. `cds-lib` is packaging-type agnostic and can be used to pull down any type of artifacts from any Maven2 compatible repository. Support of classifiers will allow devs to publish CorDapps targeting different hardware / software configurations.
+* Download a single or a range of versions from a remote repository. `corda-updates-core` is packaging-type agnostic and can be used to pull down any type of artifacts from any Maven2 compatible repository. Support of classifiers will allow devs to publish CorDapps targeting different hardware / software configurations.
 
-`cds-lib` will support pluggable transports:
+`corda-updates-core` will support pluggable transports:
 * *HTTP(s)*. Available in Maven Resolver out-of-the-box with proxy- and repository- level authentications support.
 * *Corda Flows*. Transport implementation that allows transferring files over Corda Flows.
-* *Corda RPC*. Its essentially the same as *Corda Flows*, with the difference that the *RPC* version will be used if `cds-lib` is invoked from outside of a Corda node.
 
-`cds-lib` will be invokable from a command line, with command line parameters taking precedence over the configuration file. CLI invocation *could* look like:
+`corda-updates-core` will be invokable from a command line, with command line parameters taking precedence over the configuration file. CLI invocation *might* look like:
 
 ```
-# Will pull down a single version of cordapp and its dependencies
-> java -jar cds-lib.jar
-    -Dcordapp="com.my.company.name:corda-name:1.0"
-    -DremoteRepo="https://my-company-repo.com"
-    -DlocalRepo="/path/to/my/local/repo"
-    -Dtransport="rpc"
-    -Dconfiguration="config.properties"
-
 # Will pull down all versions of cordapps and their dependencies
-> java -jar cds-lib.jar
+> java -jar corda-updates-core.jar
```

Comment:
We have a standard way to do CLIs now, see Tommy's talk from today or ask Anthony.